### PR TITLE
Fix the broken help and code evaluation

### PR DESCRIPTION
**Pull request by SqueakyBeaver**
A few things changed with this.

1. Hidden commands will now show if you type `.help <command>` but will not show in the default help (just `.help`)
2. The commands that previously checked if someone was permitted (only JohnkaS was) now should use the `commands.is_owner()` check
3. Fixed the `.eval` command by changing how the code is cleaned up

Also, I think my formatter changed all the `'`s to `"`s because I don't remember purposefully changing that

### CODE REVIEW

**Comment by SqueakyBeaver** [comment]
The [latest commit](https://github.com/Yuhanun/Better-C-Bot/pull/33/commits/340dbd2e6c1449689e3297c9ecd8d6de4b52218f) was me adding same files for the message reposting btw

File: src/cogs/admin.py [hunk 1]
```diff
@@ -13,42 +13,58 @@ def __init__(self, bot):
...
         for cog in self.bot.user_cogs:
             self.bot.reload_extension(cog)
 
-        await ctx.send("Updated cogs:\n```\n{}\n```".format('\n'.join(self.bot.user_cogs)))
+        await ctx.send(
+            "Updated cogs:\n```\n{}\n```".format("\n".join(self.bot.user_cogs))
+        )
 
     @commands.command(hidden=True)
+    @commands.is_owner()
```
Comment (Yuhanun) [inline src/cogs/admin.py]: but then remove this?...

File: src/cogs/admin.py [hunk 2]
```diff
@@ -13,42 +13,58 @@ def __init__(self, bot):
...
-            return '\n'.join(content.split('\n')[1:-1])
+        if content.startswith("```") and content.endswith("```"):
+            return "\n".join(content.split("\n")[1:-1])
 
         # remove `foo`
-        return content.strip('` \n')
+        return content.strip("` \n")
 
     @commands.command(hidden=True)
+    @commands.is_owner()
```
Comment (Yuhanun) [inline src/cogs/admin.py]: Same here? im not sure if im owner even.

**Review by Yuhanun — CHANGES_REQUESTED** [review]

File: src/cogs/admin.py (hunk 1 above)
Comment (SqueakyBeaver) [reply src/cogs/admin.py]: Should be fixed with the latest commit

File: src/cogs/admin.py (hunk 2 above)
Comment (SqueakyBeaver) [reply src/cogs/admin.py]: Latest commit should fix this

**Review by Yuhanun — APPROVED** [review]

**Pull request merged**